src/core/server/base.js

```diff
@@ -259,16 +273,19 @@ export function runServer({ listen = true, app = appName } = {}) {
         const routes = require(`${app}/routes`).default;
         const createStore = require(`${app}/store`).default;
         /* eslint-enable global-require, import/no-dynamic-require */
-        const server = baseServer(routes, createStore, { appInstanceName: app });
+        const server = baseServer(
+          routes, createStore, { appInstanceName: app });
         if (listen === true) {
           server.listen(port, host, (err) => {
             if (err) {
               reject(err);
             }
-            log.info(`🔥  Addons-frontend server is running [ENV:${env}] [APP:${app}] ` +
-                     `[isDevelopment:${isDevelopment}] [isDeployed:${isDeployed}] ` +
-                     `[apiHost:${config.get('apiHost')}] [apiPath:${config.get('apiPath')}]`);
-            log.info(`👁  Open your browser at http://${host}:${port} to view it.`);
```

Comment:
We allow 100 character lines elsewhere so this line and some of the others could probably stay as they were.